Workflow/Modules/ModuleBase.py

```diff
@@ -357,7 +357,7 @@ def setApplicationStatus( self, status, sendFlag = True ):
       # The application status won't be updated in case the workflow or the step is failed already
       if not isinstance( status, basestring ):
         status = str( status )
-      self.log.verbose( 'setJobApplicationStatus(%d, %s)' % ( self.jobID, status ) )
+      self.log.verbose( 'setJobApplicationStatus(%s, %s)' % ( self.jobID, status ) )
```

Comment:
I mean just that "%d" % 5 and "%s" % 5 will give the same result. So, %s here covers both cases